Games/types/Mafia/roles/cards/KillImmune.js

```diff
@@ -0,0 +1,13 @@
+const Card = require("../../Card");
+
+module.exports = class KillImmune extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.immunity.kill = 1;
+        this.cancelImmunity = ["lynch"];
+        this.cancelImmunity = ["lynch", "consume"];

```

Comment:
not compatible with our code

Reply:
it's currently written as an effect

further more, this makes granny edible by ww